content/usecases.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Use Cases"
+date: 2018-10-08T00:00:00-04:00
+draft: false
+layout: faq
+description: 
+---
+
+                                                                                                                                                                                                                                                                                                                                                                             
+-------------------------------
+Cool uses for qri: Publish, Sync, Collaborate, Inter-operate & Audit
+
+
+**Publish**
```

Comment:
I think we should replace the feature-driven headlines with field-specific ones. So this example could be  "sports stats"